tests/Database/StatementTest.php

```diff
@@ -247,6 +247,16 @@ public function testToArray(): void
         );
     }
 
+    public function testToObj(): void
+    {
+        $table = $this->database->table('sample_table');
+        $this->fillData();
+
+        $result = $table->limit(1)->fetchAll(\PDO::FETCH_OBJ);
```

Comment:
Maybe we should introduce new constant in StatementInterface for this fetch mode?
https://github.com/spiral/database/blob/ef943107537277e3995017982deeb6cd310093ca/src/StatementInterface.php#L24

Otherwise it may be overusing underlying details about bypassing provided mode to PDO.

Reply:
Agreed abouyt that.